src/pgo_pool.erl

```diff
@@ -97,6 +98,16 @@ init({Pool, PoolConfig}) ->
     Codel1 = start_idle(Now, Now, start_poll(Now, Now, Codel)),
     {ok, {busy, QueueTid, Codel1}}.
 
+normalize_pool_config(PoolConfig) when is_list(PoolConfig) ->
+    maps:from_list(PoolConfig);
```

Comment:
Shouldn't this be: `normalize_pool_config(maps:from_list(PoolConfig))`? So that it normalizes the configs both when its passed as a list and as a map?

And sorry this took me almost 2 weeks!

Reply:
oh yes! that makes much more sense